src/main/java/uk/gov/mca/beacons/api/jpa/entities/BeaconNote.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.mca.beacons.api.jpa.entities;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.validation.constraints.Email;
+import lombok.Getter;
+import lombok.Setter;
+import uk.gov.mca.beacons.api.domain.NoteType;
+
+@Entity
+@Getter
+@Setter
+public class BeaconNote {
```

Comment:
Could we name this as just `Note` rather than `BeaconNote`?

We have defined `Use` as `BeaconUse` when I don't think it's actually needed and have updated `BeaconPerson` to be just `Person` so thinking we could just name this as `Note`?